src/main/java/com/plaid/client/model/paymentinitiation/PaymentOptions.java

```diff
@@ -0,0 +1,34 @@
+package com.plaid.client.model.paymentinitiation;
+
+public class PaymentOptions {
+  private boolean requestRefundDetails;
+  private Bacs bacs;
+  private String iban;
+
+  public PaymentOptions withRefundDetails(boolean requestRefundDetails) {
```

Comment:
i think it might be a little clearer to call this `PaymentCreateOptions` since we could in the future add options for say PaymentGet that differ from PaymentCreate

Reply:
I've updated the class name